README.md

```diff
@@ -376,11 +367,9 @@ Annotated Datadog config JSON:
 }
 ```
 
-You also need to provide a JSON-formatted text config file that sets options for the Datadog tracing.
-
 ### Tracing Envoy & Istio
 
-Coming soon!
+See: https://docs.datadoghq.com/integrations/envoy/
```

Comment:
That's for metrics. Correct URL: https://docs.datadoghq.com/tracing/proxies/envoy/
I'd forgotten to go back and update it in the README

Reply:
lol oops